lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/REEFLauncher.java

```diff
@@ -47,114 +48,150 @@
 import java.util.logging.Logger;
 
 /**
- * The main entrance point into any REEF process. It is mostly reading from the command line to instantiate
+ * The main entry point into any REEF process (Driver and Evaluator).
+ * It is mostly reading from the command line to instantiate
  * the runtime clock and calling .run() on it.
  */
 public final class REEFLauncher {
 
   /**
-   * Parameter which enables profiling.
+   * Parameter to enable profiling. By default profiling is disabled.
    */
   @NamedParameter(doc = "If true, profiling will be enabled", short_name = "profiling", default_value = "false")
-  public static final class ProfilingEnabled implements Name<Boolean> {
-  }
+  private static final class ProfilingEnabled implements Name<Boolean> { }
 
   private static final Logger LOG = Logger.getLogger(REEFLauncher.class.getName());
 
-  private static final Configuration LAUNCHER_STATIC_CONFIG = Tang.Factory.getTang().newConfigurationBuilder()
-          .bindSetEntry(Clock.RuntimeStartHandler.class, PIDStoreStartHandler.class)
-          .bindNamedParameter(RemoteConfiguration.ErrorHandler.class, REEFErrorHandler.class)
+  private static final Tang TANG = Tang.Factory.getTang();
+
+  private static final Configuration LAUNCHER_STATIC_CONFIG =
+      TANG.newConfigurationBuilder()
           .bindNamedParameter(RemoteConfiguration.ManagerName.class, "REEF_LAUNCHER")
+          .bindNamedParameter(RemoteConfiguration.ErrorHandler.class, REEFErrorHandler.class)
           .bindNamedParameter(RemoteConfiguration.MessageCodec.class, REEFMessageCodec.class)
+          .bindSetEntry(Clock.RuntimeStartHandler.class, PIDStoreStartHandler.class)
           .build();
 
   static {
     LoggingSetup.setupCommonsLogging();
   }
 
-  private final String configurationPath;
+  /** If true, turn on Wake profiling. */
```

Comment:
Minor note: changing the name to `isWakeProfilingEnabled` would eliminate the need for the comment.